xml/en/docs/community.xml

```diff
@@ -0,0 +1,136 @@
+<!--
+  Copyright (C) F5, Inc.
+  -->
+
+<!DOCTYPE article SYSTEM "../../../dtd/article.dtd">
+
+<article name="Welcome to the NGINX Community!"
+         link="/en/docs/community.html"
+         lang="en"
+         rev="1">
+
+
+<section>
+
+<para>
+Open source powers much of the technology world.
+NGINX has been a strong part of this movement from its inception.
+As the internet has grown, so have we, from our initial web server to the
+myriad of projects involving JavaScript, Kubernetes, API gateways,
+OpenTelemetry, and WebAssembly.
+NGINX is committed to open source both in ethos and practicality, aiming to
+deliver reliable and cutting-edge technology within a healthy open source
+culture.
+Whether NGINX is part of your open source past or a step in your present,
+we welcome you into the future together.
+</para>
+
+</section>
+
+
+<section name="Mission">
+
+<para>
+We believe that healthy communities are core to innovation and open source
+sustainability.
+It is our mission to enable the NGINX community to engage with
+and develop solutions based on open and available NGINX technologies.
+Together we can power the future.
+</para>
+
+<para>
+Our community is your entry into an open world of information, collaboration,
+and discussion.
+Thanks for joining us.
+</para>
+
+</section>
+
+
+<section name="Code of Conduct">
+
+<para>
+Per our values, NGINX strives to create an inclusive community for all.
+Please remember our
+<link url="https://github.com/nginxinc/.github/blob/main/CODE_OF_CONDUCT.md">
+Code of Conduct</link> (CoC) when participating in the NGINX community.
+</para>
+
+<para>
+If you wish to tell us about a CoC violation, please email us here:
+ <link url="mailto://nginx-oss-community@f5.com">nginx-oss-community@f5.com
+</link>
+</para>
+
+</section>
+
+
+<section name="Community Spaces">
+
+<para>
+Find the NGINX community here:
+
+<list type="bullet">
+
+<listitem>
+<link url="https://twitter.com/nginxorg">Twitter</link>
+</listitem>
+
+<listitem>
+<link url="https://www.youtube.com/nginxinc">YouTube</link>
+</listitem>
+
+<listitem>
+<link url="/en/docs/governance.html">Governance</link>
```

Comment:
This link isn't somewhere you find the NGINX community. There seems no reason for it to be here, especially when it is prominently linked-to from the support page (see general comment about that).

I suggest replacing this item with a link to the NGINX community Slack.